platform/servicecontrol.include.md

```diff
@@ -10,6 +10,10 @@ To enable [ServiceControl](/servicecontrol) to gather this information, configur
  * configure [recoverability](/nservicebus/recoverability) to store information on messages failures;
  * [install plugins on the endpoints](/servicecontrol/plugins/) to monitor their health and sagas and use custom checks.
 
+NOTE: All endpoints should be configured to forward to the same audit, error, and ServiceControl plugin queues unless the system has been sharded between multiple ServiceControl instances.
```

Comment:
Is there any doco on sharding? It might be better to just unilaterally tell people to send everything to the same queues. If people make the decision to shard then it is up to them to ignore this rule (or this comment could be moved to that doco)